package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -476,6 +477,7 @@ dhcp_add() {
 	config_get leasetime "$cfg" leasetime 12h
 	config_get options "$cfg" options
 	config_get_bool dynamicdhcp "$cfg" dynamicdhcp 1
+	config_get tag_list "$cfg" tags
```

Comment:
Yes this looks better.
I asked for this change to prevent possible issues with tags containing white spaces; if a tag contains a white space it will be interpreted as different tags with a for loop.
Using config_list_foreach a tag can be configured having one or more white spaces.